src/shaper_srv.erl

```diff
@@ -126,7 +126,7 @@ handle_call({wait, Host, Action, FromJID, Size},
             From, State=#state{max_delay=MaxDelayMs}) ->
     Key = new_key(Host, Action, FromJID),
     Shaper = find_or_create_shaper(Key, State),
-    State1 = update_access_time(Key, erlang:timestamp(), State),
+    State1 = update_access_time(Key, erlang:system_time(), State),
```

Comment:
This for example is knowledge of when a shaper was accessed for the last time, and it's local knowledge to the running system, so `erlang:system_time()` is enough.